packages/core/src/RPA/core/windows/helpers.py

```diff
@@ -7,9 +7,12 @@
     import psutil
 
 
-def get_process_list() -> Dict:
+def get_process_list(get_process_info: bool = True) -> Dict:
     """Get process list.
 
     Returns dictionary mapping process id to process name
     """
-    return {proc.pid: proc.name() for proc in psutil.process_iter()}
+    if get_process_info:
+        return {proc.pid: proc.name() for proc in psutil.process_iter()}
+    else:
+        return {}
```

Comment:
Since this seems to have no utility when is called with `get_process_info=False` (as it returns an empty dict), maybe is better after all to not control the return content by this parameter and remove it, while controlling the data right in the caller:
```python
process_list = get_process_list() if self.ctx.list_processes else {}
```

Reply:
It is easier imho to handle return result here than to check all possible places where result is handled and change them.

Reply:
And wouldn't that require to refactor `get_process_list()` into `get_process_list(self.ctx.list_processes)` in "all possible places where result is handled"?

Also, luckily, this is used just here:
```
cmin@robocorp-mac rpaframework % find . -type d -name src -exec grep -rni "get_process_list" {} \;
./packages/core/src/RPA/core/windows/window.py:10:from RPA.core.windows.helpers import IS_WINDOWS, get_process_list
./packages/core/src/RPA/core/windows/window.py:73:        process_list = get_process_list()
Binary file ./packages/core/src/RPA/core/windows/__pycache__/helpers.cpython-310.pyc matches
Binary file ./packages/core/src/RPA/core/windows/__pycache__/window.cpython-310.pyc matches
./packages/core/src/RPA/core/windows/helpers.py:10:def get_process_list() -> Dict:
Binary file ./packages/windows/src/RPA/Windows/__pycache__/utils.cpython-310.pyc matches
./packages/windows/src/RPA/Windows/utils.py:4:from RPA.core.windows.helpers import IS_WINDOWS, get_process_list  # noqa: F401
```

Reply:
The `get_process_list()` will be having a default setting so there is no need to give the value for that.

Reply:
I was not sure if this is used only in the rpaframework packages or maybe even directly by the users / in inspector or in whatever place.